internal/graphql/otel_graphql_engine_detailed_test.go

```diff
@@ -13,7 +13,6 @@ import (
 	"github.com/TykTechnologies/graphql-go-tools/pkg/engine/resolve"
 	"github.com/TykTechnologies/graphql-go-tools/pkg/operationreport"
 	"github.com/TykTechnologies/graphql-go-tools/pkg/postprocess"
-	semconv "github.com/TykTechnologies/opentelemetry/semconv/v1.0.0"
```

Comment:
semconv shouldn't be removed, it should be used for passing attributes to spans rather than hardcoding attribute key/values.